cms/djangoapps/contentstore/views/component.py

```diff
@@ -107,7 +108,7 @@ def _load_mixed_class(category):
     Load an XBlock by category name, and apply all defined mixins
     """
     # Libraries v2 content doesn't have an XBlock.
-    if category == 'library_v2':
+    if category in ('library_v2', 'itembank'):
         return None
```

Comment:
I'm assuming this is the case for itembank too

Reply:
@DanielVZ96 unlike library_v2, itembank is actually a real XBlock tag (the class is ItemBankBlock), so it'd be fine to let this function return the mixed class.

(The only thing that this function's return value is used for is templates, which ItemBankBlock doesn't currently use, so you're change here _technically_ isn't breaking anything. Still, I'd rather you revert this line so that we can minimize the number of special cases.)

Reply:
Addressed with 7127d436553bdd67fdf89ca1752777524329ccb8